administrator/language/en-GB/com_banners.ini

```diff
@@ -77,6 +77,8 @@ COM_BANNERS_ERR_ZIP_DELETE_FAILURE="Zip delete failure"
 COM_BANNERS_ERROR_UNIQUE_ALIAS="Another Banner from this category has the same alias (remember it may be a trashed item)."
 COM_BANNERS_EXTRA="Additional Information"
 COM_BANNERS_FIELD_ALT_LABEL="Alt Text"
+COM_BANNERS_FIELD_SIZES_LABEL="Custom responsive sizes"
+COM_BANNERS_FIELD_SIZES_DESC="Responsive sizes in which you want the image to be generated."
```

Comment:
Please sort the keys alphabetical. DESC comes before LABEL. In the next file too